index2.html

```diff
@@ -7,6 +7,7 @@
     <title>Landing Page</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
     <style>
+    @import "https://yarnpkg.com/en/package/normalize.css";
       .jessie-content-grid .col{
```

Comment:
L11 through L38 should be on the same nesting level as L10, because the `@import` is sibling, not nesting any properties.

Reply:
si